config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -5,15 +5,32 @@ presubmits:
     - master
     always_run: true
     skip_report: false
+    decorate: true
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
-        args:
-        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
+      - command:
         - ./bin/test-on-prow.sh
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+      volumeMounts:
+      - mountPath: /lib/modules
+        name: modules
+        readOnly: true
+      - mountPath: /sys/fs/cgroup
+        name: cgroup
+      env:
+      - name: DOCKER_IN_DOCKER_ENABLED
```

Comment:
I doubt `./bin/test-on-prow.sh` does anything with this environment variable so it isn't necessary.